pcsx2/gui/AppConfig.cpp

```diff
@@ -883,6 +884,7 @@ void AppConfig::GSWindowOptions::LoadSave( IniInterface& ini )
 	IniEntry( IsFullscreen );
 
     IniEntry( IsToggleFullscreenOnDoubleClick );
+	IniEntry(IsToggleAspectRatioSwitch);
```

Comment:
Looks fine in VS. 

It seems to be because the indentation for FullscreenOnDoubleClick was created with spaces, unlike the other other lines which use a tab.